.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Yarn publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+        cache: 'yarn'
+    
+    - name: Install dependencies in react-components
+      run: yarn install
+    
+    - name: Build package NPM package
+      run: yarn build
+    
+    - name: Publish build artifacts to NPM
+      run: npm publish --access public --tolerate-republish
```

Comment:
It doesn't look like `npm` has `--tolerate-publish` as an option: https://docs.npmjs.com/cli/v6/commands/npm-publish

If you want to keep it, you would need to use [`yarn npm publish`](https://yarnpkg.com/cli/npm/publish)

Reply:
Good find, I will research that and see if we need the tolerate-republish or if we can just use the suggested